index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="top-section">
+        <div class="top-bar">
+            <div class="time"><img src="/images/Header icons/time.png" alt=""></div>
+            <div><img src="/images/Header icons/Cellular Connection.png" alt=""></div>
+            <div><img src="/images/Header icons/Wifi.png" alt=""></div>
+            <div><img src="/images/Header icons/Capacity.png" alt=""></div>
+        </div>
+        <div class="main-menu">
+            <div class="john">John Doe</div>
+            <div class="union"><img src="images/Header icons/Union.png" alt=""></div>
+        </div>
+    </section>   
```

Comment:
- [ ] Good job on doing this, I salute your attention, hoverer there are few things you will need to change before you go to the next project,
    - Since this is your header section, let's start by changing this section tag into a `header` tag instead.
    - Please remove the `div class top-bar`, that section should not be part of your application 😄 
    - on line 18, please change that div into a`nav` tag since it is the one holding your nav items.
    - on line 19, change that div into an `a` tag because your logo name will be used as a link to your home page
       - feel free to use your name or your proper logo, kindly replace john doe with your name or your logo design
       - then remove it from the nav tag and put it inside the header section only since it is not supposed to be one of the nav items